python/pyspark/sql/tests/streaming/test_streaming.py

```diff
@@ -315,6 +315,19 @@ def _assert_exception_tree_contains_msg_default(self, exception, msg):
             contains = msg in e.desc
         self.assertTrue(contains, "Exception tree doesn't contain the expected message: %s" % msg)
 
+    def test_query_manager_get(self):
+        df = self.spark.readStream.format("rate").load()
+        for q in self.spark.streams.active:
+            q.stop()
```

Comment:
What is this for?

Reply:
To make sure there is no other running queries for test reasons. I see this used in other places of the tests too.

Reply:
But that does not affect the test, right?